projects/br_scs/src/Mirza/BusinessRegistry/GenerateUtils.hs

```diff
@@ -22,16 +19,11 @@ dummyBusiness unique = NewBusiness (GS1CompanyPrefix ("Business" <> unique <> "P
                                    (mockURI unique)
 
 
-dummyUser :: T.Text -> GS1CompanyPrefix -> IO NewUser
-dummyUser unique business_uid = do
+dummyUser :: T.Text -> IO NewUser
+dummyUser unique = do
   let newUserOAuthSub     = "User" <> unique <> "OAuthSub"
-  let newUserEmailAddress = unsafeEmailAddress (encodeUtf8 unique) "example.com"
-  let newUserCompany      = business_uid
-  let newUserFirstName    = "User" <> unique <> "FirstName"
-  let newUserLastName     = "User" <> unique <> "LastName"
-  let newUserPhoneNumber  = "User" <> unique <> "PhoneNumber"
   pure NewUser{..}
 
 
-generateMultipleUsers :: [(T.Text, GS1CompanyPrefix)] -> IO [NewUser]
-generateMultipleUsers = traverse (uncurry dummyUser)
+generateMultipleUsers :: [T.Text] -> IO [NewUser]
```

Comment:
Similarly, probably can do with `[NewUser]`
```suggestion
generateMultipleUsers :: [T.Text] -> [NewUser]
```

Reply:
Thanks for picking this us fb44770d5e31301015d4f39b75c6507af141b209.